snippets/pagination.liquid

```diff
@@ -9,14 +9,13 @@
     Accepts:
     - paginate: {Object}
     - anchor: {String} (optional) This can be added so that on page reload it takes you to wherever you've placed your anchor tag.
-    - class: {String} (optional) Appended to container element's class attribute
```

Comment:
On `main-collection-product-grid.liquid` file, can we remove the `class` parameter as well?

![image](https://user-images.githubusercontent.com/658169/124977794-ec0e9d00-dffe-11eb-9d24-4f52c587cea1.png)

Reply:
Updated